utils/generate_backend_completeness.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import importlib
+import inspect
+from pathlib import Path
+from typing import Any
+from typing import Final
+
+import polars as pl
+from jinja2 import Template
+
+TEMPLATE_PATH: Final[Path] = Path("utils") / "api-completeness.md.jinja"
+DESTINATION_PATH: Final[Path] = Path("docs") / "api-reference" / "api-completeness.md"
+
+
+MODULES = ["dataframe", "series", "expression"]
+EXCLUDE_CLASSES = {"BaseFrame"}
+
+
+def get_class_methods(kls: type[Any]) -> list[str]:
+    return [m[0] for m in inspect.getmembers(kls) if not m[0].startswith("_")]
+
+
+def get_backend_completeness_table() -> str:
+    results = []
+
+    for module_name in MODULES:
+        nw_namespace = f"narwhals.{module_name}"
+        sub_module_name = module_name if module_name != "expression" else "expr"
```

Comment:
![WhyThoGIF](https://github.com/user-attachments/assets/d0225eea-0755-4b7e-bb3b-5323763be448)

Shall we change everything now?

Reply:
yeah let's make it expr 😎